sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -430,8 +433,9 @@ def get_tag_properties(self, repository: str, tag: str, **kwargs) -> ArtifactTag
             for tag in client.list_tag_properties("my_repository"):
                 tag_properties = client.get_tag_properties("my_repository", tag.name)
         """
+        tag_properties = self._client.container_registry.get_tag_properties(repository, tag, **kwargs)
         return ArtifactTagProperties._from_generated(  # pylint: disable=protected-access
-            self._client.container_registry.get_tag_properties(repository, tag, **kwargs),
+            tag_properties.tag, # type: ignore
```

Comment:
mypy think `tag` could be `None` though it says required in docstring: https://github.com/Azure/azure-sdk-for-python/blob/9f7a94a46062ff80009ce24ccc2836984c82692d/sdk/containerregistry/azure-containerregistry/azure/containerregistry/_generated/models/_models.py#L391

Reply:
There's a related comment above: https://github.com/Azure/azure-sdk-for-python/pull/28196#discussion_r1140720516

Reply:
mypy doesn't look at docstrings, but basically it's a required readonly property so generated code sets it to none in the init. i think i'm ok with the tag properties. when the sdk switches over to customizations instead of wrapping i think this issue will go away